packages/web-components/src/components/video-player/__stories__/video-player.stories.ts

```diff
@@ -10,22 +10,31 @@
 import { html } from 'lit-element';
 import readme from './README.stories.mdx';
 import '../video-player-container';
+import '../../lightbox-media-viewer/lightbox-video-player-container';
 
 export const Default = () => html`
-  <dds-video-player-container video-id="1_9h94wo6b"></dds-video-player-container>
+  <dds-video-player-container playing-mode video-id="1_9h94wo6b"></dds-video-player-container>
 `;
 
 export const aspectRatio1x1 = ({ parameters }) => {
   const { videoId, aspectRatio } = parameters?.props?.VideoPlayer ?? {};
   return html`
-    <dds-video-player-container video-id=${videoId} aspect-ratio=${aspectRatio}></dds-video-player-container>
+    <dds-video-player-container playing-mode video-id=${videoId} aspect-ratio=${aspectRatio}></dds-video-player-container>
```

Comment:
Did you mean to set `playing-mode` from a knob, or something else...? `inline` or `lightbox` has to be set as the attribute value.